src/Sylius/Behat/Resources/config/services/pages/shop.xml

```diff
@@ -23,6 +23,7 @@
         <parameter key="sylius.behat.page.shop.cart_summary.class">Sylius\Behat\Page\Shop\Cart\SummaryPage</parameter>
         <parameter key="sylius.behat.page.shop.product.show.class">Sylius\Behat\Page\Shop\Product\ShowPage</parameter>
         <parameter key="sylius.behat.page.shop.order.order_payments.class">Sylius\Behat\Page\Shop\Order\OrderPaymentsPage</parameter>
+        <parameter key="sylius.behat.page.shop.shop_home.class">Sylius\Behat\Page\Shop\ShopHomePage</parameter>
```

Comment:
Homepage with this service id already exists. Should i tag old homepage with legacy prefix/sufix?